lib/drivers/abbott/abbottFreeStyleLite.js

```diff
@@ -440,30 +466,36 @@ module.exports = function (config) {
         }
         data.connect = true;
         debug(result);
-        _.extend(data, _.pick(result, 'serialNumber', 'softwareVersion', 'deviceTime'));
+        _.extend(cfg.deviceInfo, _.pick(result, 'serialNumber', 'softwareVersion', 'deviceTime'));
 
-        if (data.serialNumber[0] === 'C') {
+        if (cfg.deviceInfo.serialNumber[0] === 'C') {
           // there are a wide variety of Abbott meters that start with C and D,
           // but this is the only way to distinguish the FreeStyle Freedom Lite
           // and the FreeStyle Lite as they use the same cable, so the USB PID/VID
           // is the same.
-          data.model = 'AbbottFreeStyleFreedomLite';
-        } else if (data.serialNumber[0] === 'D') {
-          data.model = 'AbbottFreeStyleLite';
+          cfg.deviceInfo.model = 'AbbottFreeStyleFreedomLite';
+        } else if (cfg.deviceInfo.serialNumber[0] === 'D') {
+          cfg.deviceInfo.model = 'AbbottFreeStyleLite';
         } else {
-          data.model = 'UnknownFreeStyle';
+          cfg.deviceInfo.model = 'UnknownFreeStyle';
         }
-        debug('Detected as: ', data.model);
+        debug('Detected as: ', cfg.deviceInfo.model);
 
-        data.id = data.model + ' ' + data.serialNumber;
+        data.id = cfg.deviceInfo.model + ' ' + cfg.deviceInfo.serialNumber;
```

Comment:
Seems like a good candidate for a template literal: 
```suggestion
        data.id = `${cfg.deviceInfo.model} ${cfg.deviceInfo.serialNumber}`;
```

Reply:
Are template literals not ES6-specific? I vaguely remember us not wanting to mix ES6 syntax into the older drivers until they're updated to ES6.

Reply:
Ah yes. Okay.